ngraph/frontend/frontend_manager/src/frontend_manager.cpp

```diff
@@ -191,24 +191,24 @@ std::vector<Place::Ptr> InputModel::get_outputs() const
 
 Place::Ptr InputModel::get_place_by_tensor_name(const std::string& tensorName) const
 {
-    FRONT_END_NOT_IMPLEMENTED(get_place_by_tensor_name);
+    return nullptr;
```

Comment:
It is virtual method which returns 'nullptr' now by default. Previously we raised an exception, but after discussion we've decided that if place is not found - it shall return nullptr. So that for clients like 'MO' coding would be easier (no need to try/except everywhere)